Tools/glib/dependencies/apt

```diff
@@ -111,6 +111,9 @@ PACKAGES=(
     git-svn
     subversion
 
+    # These are dependencies necessary for using git-webkit.
+    python-dev
```

Comment:
Use `python3-dev` instead? Ubuntu and Debian doesn't seem to have a python-dev package.

Debian: https://packages.debian.org/search?keywords=python-dev&searchon=names&exact=1&suite=stable&section=all
Ubuntu: https://packages.ubuntu.com/search?keywords=python-dev&searchon=names&exact=1&suite=all&section=all